docs/aerodromes/classc/Pearce.md

```diff
@@ -4,7 +4,69 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | Callsign       | Frequency        | Login ID              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| **Pearce ADC**    | **Pearce Tower**  | **118.300**    | **PE_TWR**        |
+| **Pearce SMC**    | **Pearce Ground**  | **127.250**   | **PE_GND**        |
+| **Pearce ACD**    | **Pearce Delivery**  | **134.100** | **PE_DEL**       |
+| Pearce ATIS    |   | 136.400         | YPEA_ATIS       |
+
+## Airsapce
+
+PE ADC owns the Class C airspace **in the PE CTR** within 5nm of the YPEA ARP from `SFC` to `A035`. This airspace is primarily used for military circuits and initial and pitch approaches.
+
+### Sid Selection
+Aircraft planned via **WAMAL**, **GUNOK**, **ALKIMOS**, **MULLALOO**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
```

Comment:
Just double checking the alkimos and Mullaloo waypoints are visual waypoints? If not, change to the waypoint names

Reply:
Curious how a visual waypoint can be attached to a procedural sid?

Reply:
While they are towns, the pilots fly it according to the tacan hence its a proc sid.

Reply:
Can I get a reference for this in the manuals?

Reply:
https://ais-af.airforce.gov.au/sites/default/files/current-cycle-products/2412%20TERMA.pdf page 143

Reply:
How I am interpreting those charts is that those vfr waypoints aren't actually fileable waypoints in terms of IFR flight plans, it is just the SID Name given to position the aircraft outside of the PEA CTA. Those fully spelt out names are just placeholders to get the aircraft to their specific exit gates.

Reply:
Ok, so they are a proc side, but should I make it a note or something?

Reply:
Go have a read of the YSRI SID Selection section and write it kinda the same. 

Have is say something aswell like "The Radar SID is preferred, however a procedural SID may be issued on request"